ViewModel/EnvironmentAndExtendStoreUuid.php

```diff
@@ -14,6 +14,12 @@
 
 class EnvironmentAndExtendStoreUuid implements \Magento\Framework\View\Element\Block\ArgumentInterface
 {
+    const EXTEND_CONFIG_ENVIRONMENT = [
+        // This is for custom mapping of Integration environments to Extend environments
+        'dev' => 'development',
+        'prod' => 'production',
```

Comment:
right now we default to `prod` if we can't find a valid env but technically env name is `production`